packages/cx-templates/doc/README.md

```diff
@@ -50,40 +52,59 @@ As they design templates, Site Builders can choose the following:
 
-## Content Editor
-
-When the content editor creates a new page, it will create a new page using the default template.
-They can go to Page | Template and choose which template they would like to switch to.  (@jones see /styleguide section and you will get the templates button)
-
-## Site Builder
-
-The choice of what templates and setting templates is done within the [cxPage](https://github.com/johnsonandjohnson/Bodiless-JS/tree/main/packages/cx-templates/src/components/Page) Component.
-
-First create a Page token defining what templates & their tokens should apply to a page option and exported as necessary.
-
-````js
-const Default = asFluidToken({
-  ...cxPage.Default,
-  Components: {
-    _default: on(GenericTemplateClean)(cxGenericTemplate.Default),
-    PDP: on(PDPClean)(cxPDP.Default),
-    PLP: on(PLPClean)(cxPLP.Default),
-  },
-});
-
-export default {
-  ...cxPage,
-  Default,
-};
-
-````
-
-Then within the _default.jsx file in the sites/NAMEOFSITE/src/templates it will call use this token.
-
-````js
-const DefaultPage = as(__cxstarter__Page.Default)(Fragment);
-
-export default DefaultPage;
-```
-
-@jones -- i think there is one more step to get the choice of Templates in the Template UI list seen by content editor -- need to fidn that step. :)
+## Content Editor Details
+
+When you create a new page, it will create the new page using the default template.
+
+### Change Template
+
+![Template icon](../assets/PageTemplateIcon.jpg ':size=60')
+
+To switch the template that a page is using:
+
+01. Navigate to the page for which you'd like to change the template.
+01. From the [Toolbar](/ContentEditorUserGuide/#toolbar), click **Page > Template** to open the
+    _Choose a template for this page_ form.  
+    !["Choose a template for this page" form](../assets/PageChooseTemplate.jpg ':size=50%')
+01. Select the radio button of the template you'd like to switch to, and click the checkmark to
+    confirm.
+01. The page template should automatically update.
+
+## Site Builder Details
+
+### Configure Available Templates
+
+The list of available templates in the _Choose a template for this page_ form is configurable via
+the
+[`cxPage`](https://github.com/johnsonandjohnson/Bodiless-JS/tree/main/packages/cx-templates/src/components/Page
+':target=_blank') Component.
+
+01. First, create a Page token defining what templates and their tokens should apply to a page
+    option, and export as necessary.
+    ```js
+    const Default = asFluidToken({
+      ...cxPage.Default,
+      Components: {
+        _default: on(GenericTemplateClean)(cxGenericTemplate.Default),
+        PDP: on(PDPClean)(cxPDP.Default),
+        PLP: on(PLPClean)(cxPLP.Default),
+      },
+    });
+
+    export default {
+      ...cxPage,
+      Default,
+    };
+    ```
+01. Within the `_default.jsx` file in the `sites/SITE-NAME/src/templates` directory, it will call
+    using this token.
```

Comment:
replace "it will call using this token" 
to 
the _default template will then call the page component created from the previous step.   This will replace the gatsby _default content with the 'page template component' and editor functionality to allow content editor to switch between template components.